packages/core/strapi/lib/services/entity-service/types/params/populate.d.ts

```diff
@@ -0,0 +1,45 @@
+import type { Attribute, Common, Utils } from '@strapi/strapi';
+
+/**
+ * Wildcard notation for populate
+ *
+ * To populate all the root level relations
+ */
+type WildcardNotation = '*';
+
+type PopulatableKeys<TSchemaUID extends Common.UID.Schema> = Utils.Guard.Never<
+  Attribute.GetPopulatableKeys<TSchemaUID>,
+  string
+>;
+
+/**
+ * Union of all possible string representation for populate
+ *
+ * @example
+ * type A = 'image'; // ✅
+ * type B = 'image,component'; // ✅
+ * type c = '*'; // ✅
+ * type D = 'populatableField'; // ✅
+ * type E = '<random_string>'; // ❌
+ */
+type StringNotation<TSchemaUID extends Common.UID.Schema> =
+  | WildcardNotation
+  | PopulatableKeys<TSchemaUID>
+  | `${string},${string}`
+  | `${PopulatableKeys}.${string}`;
```

Comment:
This may be an issue with documentation rather than this implementation, but the documentation for both [rest API](https://docs.strapi.io/dev-docs/api/rest/populate-select#population) and [entity service](https://docs.strapi.io/dev-docs/api/entity-service/populate) do not mention either of these methods. They only say "an array of attribute names".

Reply:
Same answer as above :eyes: